tests/test_other.py

```diff
@@ -11176,6 +11190,6 @@ def test_wasmfs_unistd_create(self):
     self.set_setting('WASMFS')
     self.do_run_in_out_file_test('wasmfs/wasmfs_create.c')
 
+  @with_wasmfs
   def test_wasmfs_unistd_seek(self):
```

Comment:
If a test runs in both wasmfs mode and otherwise, then I think we don't need `wasmfs` in the name - it's a general test at that point. But we can leave that for a cleanup later, that might be best.

Reply:
Added 👍